pkg/app/app.go

```diff
@@ -1661,9 +1661,16 @@ func (a *App) sync(r *Run, c SyncConfigProvider) (bool, []error) {
 
 	infoMsg := fmt.Sprintf(`Affected releases are:
 %s
+
+Do you really want to sync?
+  Helmfile will sync all your releases, as shown above.
```

Comment:
Seeing how it works for `apply` today, this question is printed only when the interactive mode is enabled.

Almost certainly we should follow the same pattern?

The key difference is, in `apply`, we have two variables- one for `infoMsg` and another for `confMsg` and we print `infoMsg` for the non-interactive case and `confMsg` for the interactive case.

Reply:
ok. I will see it.